install/kubernetes/helm/istio/values-istio-example-sds-vault.yaml

```diff
@@ -0,0 +1,27 @@
+global:
+  controlPlaneSecurityEnabled: false
+
+  mtls:
+    # Default setting for service-to-service mtls. Can be set explicitly using
+    # destination rules or service annotations.
+    enabled: true
+
+  # Default is 10s second
+  refreshInterval: 1s
+
+  sds:
+    enabled: true
+    udsPath: "unix:/var/run/sds/uds_path"
+    useNormalJwt: true
+
+nodeagent:
+  enabled: true
+  image: node-agent-k8s
+  env:
+    CA_ADDR: "https://35.233.249.249:8200"
+    CA_PROVIDER: "VaultCA"
+    VAULT_ADDR: "https://35.233.249.249:8200"
```

Comment:
does this have to be a real ip?

Reply:
This has to be the IP of the testing Vault server.

Reply:
Added a comment for the IP address.